apis/validator/beacon-node-oapi.yaml

```
@@ -290,6 +316,40 @@ components:
           format: uint64
           nullable: true
           description: "The slot in which a validator must propose a block, or `null` if block production is not required."
+    Validator:
+      type: object
+      properties:
+        validator_pubkey:
+          $ref: '#/components/schemas/pubkey'
+        withrawal_credentials:
+          type: string
+          format: bytes
+          pattern: "^0x[a-fA-F0-9]{64}$"
+          description: "Hash of withdrawal credentials"
+        effective_balance:
+          type: integer
+          format: int64
```

why `int` instead of `uint`?

```suggestion
          format: uint64
```

---

oApi defines only int64 type: https://swagger.io/docs/specification/data-models/data-types/#numbers

Do we wan't to define uin64 as string type to avoid rounding in various tools?

---

hmmm uint64 is defined in a number of fields above. 
It'd be a nice to have. What do you recommend?

---

We can leave like this for now. I've opened [issue](https://github.com/ethereum/eth2.0-APIs/issues/9) to gather more feedback.